scss/inc/_navigatorFizzy.scss

```diff
@@ -0,0 +1,312 @@
+        background: $backgroundColor;
+    }
+
+    /* header */
+    & > .qti-navigator-label {
+        display: flex;
+        justify-content: space-between;
+        margin: 12px 10px 0 10px;
+        padding-bottom: 10px;
+        border-bottom: 1px solid #ccc;
+
+        .qti-navigator-text {
+            @include font-size(20);
+            font-weight: bold;
+        }
+
+        .icon-close {
+            @include font-size(28);
+            line-height: $lineHeight;
+            color: $colorTextDefault;
+
+            &:hover {
+                color: $colorTextDefault;
+                text-decoration: none;
+                cursor: pointer;
+            }
+        }
+    }
+
+    /* tree */
+    .qti-navigator-tree {
+        padding: 0 10px;
+        background: none;
+
+        span.qti-navigator-label {
+            margin: 13px 0 10px 0;
+            display: block;
+            @include font-size(15);
+        }
+    }
+
+    .qti-navigator-item {
+        height: $hitboxSize;
+        width: $hitboxSize;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        /* reset parent style */
+        padding: 0 !important; /* instead of rewriting 6-classes selector on focusing */
+        border: none !important; /* instead of rewriting 6-classes selector on focusing */
+
+        &.disabled {
+            background-color: unset !important;
+            color: $colorTextDisabled !important;
+            border-color: $colorTextDisabled !important;
+            opacity: 1 !important;
+            text-shadow: none !important;
+        }
+
+        /* reset parent styles */
+        &.disabled .step {
+            background-color: $colorBgDefault !important;
+            border-style: solid !important;
+            border-radius: $radiusCircular !important;
+            border-width: $borderMedium !important;
+            color: $colorTextDisabled !important;
+            border-color: $colorTextDisabled !important;
+            opacity: 1 !important;
+            text-shadow: none !important;
+        }
+
+        /* remove number from item button if item was marked for review */
+        &.flagged span.step-label {
+            display: none;
+        }
+
+        &.viewed .step {
+            border-width: $borderMedium;
+            color: $colorTextDefault;
+            border-color: $colorTextDefault;
+        }
+
+        &.answered .step {
+            background-color: $colorBgInverted;
+            color: $colorTextInverted;
+            border-color: $colorBgInverted;
+        }
+
+        & .indicator {
+            display: none;
+        }
+
+        &.active .indicator {
+            position: absolute;
+            display: block;
+            z-index: 1;
+            color: $colorTextDefault;
+            height: 1.4rem;
+            min-width: 1.6rem;
+            background-color: $backgroundColor;
+            top: unset;
+            bottom: -1.6rem;
```

Comment:
The value seems a little off (it's clipping on the button). With -1.7rem or less, it's ok.
![Screenshot 2021-11-05 at 09 33 37](https://user-images.githubusercontent.com/43652944/140481877-1ed060fb-ddd9-45f2-994c-e6981dd86e78.png)

Reply:
Updated